l3kernel/l3tl.dtx

```diff
@@ -3061,7 +3061,7 @@
 %   }
 % \begin{macro}{\@@_trim_mark:}
 %   Trimming spaces from around the input is done using delimited
-%   arguments and quarks, and to get spaces at odd places in the
+%   arguments and \cs{@@_trim_mark:}, and to get spaces at odd places in the
```

Comment:
Compared to when dc98b75d (\tl_trim_spaces:n trimming recursively, only explicit spaces., 2011-08-13) added this doc, current implementation uses no quarks.